docs/cql/ddl.rst

```diff
@@ -289,6 +289,16 @@ For instance::
 
 The supported options are the same as :ref:`creating a keyspace <create-keyspace-statement>`.
 
+ALTER tablets-enabled KEYSPACE :label-caution:`Experimental`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Modifying a tablets-enabled keyspace is possible and there is no special CQL syntax introduced for that purpose, because in case a keyspace has tablets, altering it follows the same process as with vnode keyspaces, though there are some caveats:
+
+- the replication factor can be increased or decreased by at most 1 at a time - that being said, one can repeatedly modify RF to reach the desired value,
+- ``ALTER`` statement doesn't fully support `replication_factor` tag specified for `NetworkTopologyStrategy`, and it's safer to list DCs explicitly when altering a keyspace,
+- if there's any other ongoing global topology operation, executing ``ALTER`` statement will fail (with an explicit and specific error) and needs to be repeated,
+- ``ALTER`` statement may take longer than the regular query timeout, and if it times out, it'll be continued nevertheless.
```

Comment:
s/be continued/continue to execute in the background/

Reply:
Also, we can add that only NetworkTopologyStrategy is supported with tablets and it cannot be ALTERed.

Reply:
It's deprecated but its rejection is still configurable to allow users some grace time to adapt their apps.
See https://github.com/scylladb/scylladb/issues/5224, https://github.com/scylladb/scylladb/issues/8586

Reply:
Applied the suggestion and additional remark.